numba/typing/arraydecl.py

```diff
@@ -159,9 +159,9 @@ def check_contiguity(outer_indices):
     return Indexing(idx, res, advanced)
 
 
-@infer
+@infer_global(operator.getitem)
 class GetItemBuffer(AbstractTemplate):
-    key = "getitem"
+    key = operator.getitem
```

Comment:
Is it necessary to set the `key` attribute if you are using `infer_global`?

Reply:
I don't think so.  Let me try